src/WTG.Analyzers.Test/TestData/PointlessOverrideAnalyzer/Sealed/Source.cs

```diff
@@ -0,0 +1,140 @@
+using System;
+
+namespace Magic
+{
+	class SealedOverrideClass : BaseClass
+	{
+		public sealed override void Method(int argument1, int argument2)
+		{
+			base.Method(argument1, argument2);
+			Foo();
```

Comment:
delete. calling foo here would invalidate what you are trying to assert.

Reply:
that's what I get for copy-pasting an existing test as a base 🤦‍♂️